testing/citest/tests/azure_smoke_test.py

```diff
@@ -0,0 +1,242 @@
+#copyright 2017 Veritas Technologies, LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Smoke test to see if Spinnaker can interoperate with Microsoft Azure.
+
+See testable_service/integration_test.py and spinnaker_testing/spinnaker.py
+for more details.
+
+The smoke test will use the spinnaker configuration parameters from the server
+endpoint (gate) to determine the managed project it should verify, and to
+determine the spinnaker account name to use when sending it commands.
+
+Note:
```

Comment:
Can you link to the instructions for installing and configuring the azure CLI in the testing/citest/readme.md (which in turn could link to Microsoft pages)